File: Python/Gate Three/dayThreeFunction.py
def get_the_largest(numbers,second_numbers,third_numbers):
    largest = 0
    for outer_loop in range(len(numbers)):
 
            if(numbers[outer_loop] > largest):
                largest = numbers[outer_loop] 
  
    for index in range(len(second_numbers)):     
            if (second_numbers[index] > largest):
                largest = second_numbers[index] 

    
    for count in range(len(third_numbers)):     
            if (third_numbers[count] > largest):
                largest = third_numbers[count] 

    return largest

    

File: Java/Gate Three/DayThreeFunction.java
public class DayThreeFunction{

    public static double testTheLargest(double[][] numbers){
     
        double largest = 0;           

    for(int outerArray = 0;outerArray < numbers.length;outerArray++){
        for(int innerArray = 0;innerArray < numbers.length;innerArray++){
    
        if(numbers[outerArray][innerArray] > largest){
             largest = numbers[outerArray][innerArray];
        }
       
        }
    
     
        }
return largest;
    }




//     
//
//    public static String getNumberOfOccurence(String character){
//
//    int counter = 0;
//    for(int count = 0;count < character.length();count++){
//
//        
//    char letters =  character.charAt(count);
//
//    if(letters <= 'A' || letters >= 'Z' &  letters <= 'a'  || letters >= 'z' ){
//
//    counter ++;
//
//
//    System.out.println(letters +"" +counter);
//    }
//        
//
//
//
//
//
//
//    }
//
//



//    }

    }

















 
